js/runtime.js

```diff
@@ -572,16 +566,40 @@
         },
 
         shape: {
-            fillShape: function(shp){
-                shp();
+            fill: function(pathSet){
```

Comment:
Shouldn't the argument here be called `shape`? Also, should it be ignored in the function as it is? (this counts for `stroke` as well).